examples/with-solana-passkeys/src/pages/_app.tsx

```diff
@@ -4,7 +4,7 @@ import Head from "next/head";
 import { TurnkeyProvider } from "@turnkey/sdk-react";
 
 const turnkeyConfig = {
-  apiBaseUrl: process.env.NEXT_PUBLIC_BASE_URL!,
+  apiBaseUrl: process.env.NEXT_PUBLIC_TURNKEY_API_BASE_URL!,
```

Comment:
Great catch! I think in most env var files (e.g. `.env.local.example`) we call this `NEXT_PUBLIC_BASE_URL`, so it might be easier to swap the name in `.env.local.example` for consistency across the repo?